src/main/java/com/cinema/service/impl/mapper/CinemaHallMapper.java

```diff
@@ -0,0 +1,19 @@
+package com.cinema.service.impl.mapper;
+
+import com.cinema.model.CinemaHall;
+import com.cinema.model.dto.CinemaHallRequestDto;
+import com.cinema.model.dto.CinemaHallResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallMapper {
+
+    public CinemaHall mapDtoToCinemaHall(CinemaHallRequestDto cinemaHallRequestDto) {
+        return new CinemaHall(cinemaHallRequestDto.getCapacity(),
```

Comment:
"In your mappers prefer setters over big constructors while converting entity to dto and vice versa." same in other cases

Reply:
Quite judgemental - what is big and what isn't. I thought this case is not big... Let's see what mentor says